debian/rules

```diff
@@ -7,6 +7,11 @@ PYTHON_MINOR := $(shell $(PYTHON) -c 'import sys; print(sys.version_info[1])')
 PYTHON_FROM := $(PYTHON_MAJOR).$(PYTHON_MINOR)
 PYTHON_TO := $(PYTHON_MAJOR).$(shell echo $$(( $(PYTHON_MINOR) + 1 )))
 
+# In case of using built-in python version (which is expected to have at least 3.12)
+# we don't care about this dependency and it can be removed at all later.
+ifeq ($(PYTHON_MINOR),12)
```

Comment:
not sure how to avoid this dependency more elegant than that, maybe we can just set `SUBSTVARS` to an empty string for example

with built-in python we don't need this (as well as `Build-Depends`) - but we can clean it up later somehow

Reply:
I suppose we can workaround lack of numeric comparison in debin rules using `shell` directive.

Reply:
changed for using shell and proper comparison I guess

while this currently is needed for all builds with custom python, we can assume that we'll use it only for 3.12+, otherwise need to think how to remove this dependency properly

Reply:
Nevertheless, I think that checking environment variable here if embedded python is used and using default value `SUBSTVARS := -Vpython:Depends=""` are more clear